experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+									<defaults>
+										<default valuesource="fieldvalue" field="name" enclosevalueinquotes="true" value="cid"/>
+										<default valuesource="fieldvalue" field="type" enclosevalueinquotes="true" value="String"/>
+										<default valuesource="modelmerge" field="value" enclosevalueinquotes="true" sourcemodel="Contact" sourcefield="Id"/>
+									</defaults>
+								</action>
+								<action type="action-sequence" action-sequence-id="29f52636-5af5-49b1-8550-54170e17e336">
+									<inputs>
+										<input name="model" value="FlowParams"/>
+										<input name="flowName" value="HelloWorldFlow"/>
+									</inputs>
+								</action>
+							</actions>
+							<tooltip position="top"/>
+							<renderConditions logictype="or"/>
+							<enableConditions logictype="and"/>
+							<styleVariantConditions/>
+						</skuid__button>
+					</buttons>
+				</skuid__buttonGroup>
+			</groups>
+			<styles>
+				<spacing bottom="2"/>
+			</styles>
+			<renderConditions logictype="and"/>
+			<styleVariantConditions/>
+		</skuid__buttonSet>
+	</components>
+	<resources>
+		<labels/>
+		<javascript>
+			<jsitem location="inline" name="EventHotFix" cachelocation="false" url="">((skuid) =&gt; {
+    window.skuid.model = {};
+	window.skuid.component = {};
+    window.skuid.model.Model = Object;
+    window.skuid.component.Component = Object;
+    console.log(8);
+})(skuid)</jsitem>
+			<jsitem location="inlinesnippet" name="callFlow" cachelocation="false">const params = arguments[0],
+    eventName = "showModalFlow",
+    flowName = params.$Input.flowName,
+    model = skuid.$M(params.$Input.model);
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: flowName,
+        params: model.getRows()
+    };
+
+// If there is a matching global application event with this name,
+// then fire that event, with each Skuid Event parameter
+// being mapped directly to that Event parameter.
+if (auraEvt) {
+    if (paramsObject) {
+        auraEvt.setParams(paramsObject);
+    }
+    auraEvt.fire();
+}
+
+skuidAuraEvt.setParams({
+    "name": eventName,
+    "source": "skuid",
+});
+
+// ALSO fire a generic "Skuid Event",
+// where all of the parameters are serialized to JSON
+if (paramsObject) {
+    skuidAuraEvt.setParam("data", paramsObject);
+}
+
+skuidAuraEvt.fire();</jsitem>
+			<jsitem location="inlinesnippet" name="showVariable" cachelocation="false">const params = arguments[0];
+const eventName = "showModalFlow";
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: "Civilian_Training_Form",
```

Comment:
Two suggestions:

1. Make `flowName` a variable up at the top of this snippet similar to what you did with `eventName` on line 118
2. Make the name of this event something more generic like `MyFlowName` so that's more obvious it needs to be changed to the _actual_ name of the Flow